src/components/EquippedCharacterSheetSlot/index.tsx

```diff
@@ -14,31 +19,68 @@ interface AttributeProps {
   attributesImpacted: AttributeImpact[];
 }
 
+const POWER_TYPE_TO_COLOR: { [key in SlotPrimaryType]: string } = {
+  active: COLORS.backgroundTypeActive,
+  instant: COLORS.backgroundTypeInstant,
+  weapon: COLORS.backgroundTypeWeapon,
+  armor: COLORS.backgroundTypeArmor,
+  general: COLORS.background,
+  racial: COLORS.background,
+};
+
 const EquippedCharacterSheetSlot = (props: Props) => {
-  if (props.slot.equipped[0] === undefined) {
-    throw new Error(
-      `Nothing equipped in this slot you're trying to look at (slotNumber: ${props.slot.metadata.slotNumber})`
-    );
-  }
+  const { unequipCardFromSlot } = useContext(GameContext);
+  const { value: isModalOpen, setOff: closeModal, toggle: toggleModal } = useFlipFlop(false);
+
   /**
-   * TODO support multiple cards in a slot
+   * TODO support displaying multiple cards in a slot
    */
-  const equippedCardData = getPowerByName(props.slot.equipped[0]);
+  const equippedCards = useMemo(
```

Comment:
We don't actually need this bit, I stopped here because I'm not sure how we should display multiple cards in a slot yet, but leaving it here for later